concepts/architecture/index.md

```diff
@@ -300,23 +305,21 @@ Committee selection is dynamic and stake-based, with a new committee elected for
 
 The consensus protocol makes use of:
 
-- The communication layer for consensus round messaging and consensus state synchronisation.
-- The Autonity Protocol Contract for committee selection logic.
+- The [communication layer](/concepts/architecture/#communication-layer) for consensus round messaging and consensus state synchronisation in a [dedicated consensus channel](/concepts/system-model/#p2p-networking-protocols).
+- The [Autonity Protocol Contract](/concepts/architecture/#autonity-protocol-contract) for [committee selection logic](/reference/api/aut/op-prot/#computecommittee).
```

Comment:
I think this sentence is fine, no need to modify. However as a note now the computeCommittee logic is in a precompiled contract (although it is still called from the AC, so I think it is an implementation detail).

Reply:
OK. Resolving.